### DISCUSSION TITLE
Feature request - Group claims

### DISCUSSION THREAD

**Discussion by Kitsuneko111** [GriefPrevention/GriefPrevention#759] [Ideas]
Similar to the Admin claims, players should be able to create groups that they can add into or take away from so that towns/cities can be managed by a group rather than fed into one person

**Comment by RoboMWM** [comment]
I guess you missed the feature request template, would've helped you define your feature request better. (I'm not closing or anything but would help you better define the feature for others). If it's what I think it is I may have been thinking of making a similar addon for my own server. But idk.

**Comment by Kitsuneko111** [comment]
yes, i did miss the template. i'll look for it now ^.^
